index.html

```diff
@@ -0,0 +1,164 @@
+    <a href="/"><h3 class="mobile">MuziQ</h3></a>
+  </nav>
+    <nav class="desktop-nav">
+          <div class="first-menu">
+              <ul class="first-menu-list">
+                  <li class="list-item">
+                      <a href="#"> <img src="./images/facebook.png"  class="social" alt="facebook"> </a>
+                  </li>
+                  <li class="list-item">
+                      <a href="#"><img src="./images/twitter.png" class="social" alt="twitter"></a>
+                  </li>
+                  <li class="list-item">
+                      <a href="#">English</a>
+                  </li>
+                  <li class="list-item">
+                      <a href="#">My Page</a>
+                  </li>
+                  <li class="list-item">
+                      <a href="#">Logout</a>
+                  </li>
+              </ul>
+          </div></nav>
+          <nav class="second-dt-nav">
+          <div class="second-menu-div">
+            <div class="menu-item-brand">
+                     <a href="./index.html" class="brand">MuziQ</span></a>
+            </div>
+              <ul class="second-menu-list">
+                  <li class="menu-item">
+                      <a href="./index.html#main">Program</a>
+                  </li>
+                  <li class="menu-item">
+                      <a href="./index.html#feature">Speakers</a>
+                  </li>
+                  <li class="menu-item">
+                      <a href="./about.html">About</a>
+                  </li>
+                  <li class="menu-item get-ticket">
+                      <a href="#">Get Ticket</a>
+                  </li>
+              </ul>
+          </div>
+      </nav>
+        <nav class="mobile-nav">
+          <a href=""><span class="close-button">&#10006;</span></a> 
+           <ul class="menu-list-mobile" >
+            <li ><a class="menu-item" href="#main">Program</a></li><br>
+            <li ><a class="menu-item" href="./index.html#feature">Speakers</a></li><br>
+            <li ><a class="menu-item" href="about.html">About</a></li><br>
+            <li ><a class="menu-item" href="#">Get ticket</a></li>
+           </ul>
+        </nav>
+  </header>
+      <section class="hero-container">
+        <div class="container">
+         <div class="intro">
+          <p class="primary-intro">"Hello! Music Lovers!!</p>
+            <h1> GLOBAL Music Concert</h1>
+          </div>
+          <p class="sub-theme">MuziQ's Global Concert brings together the community of experts, creators
+              and producers who compose the good and quality music we all love
+              to hear. This year, we hope to expand our broadcast by streaming on multiple platforms. 
+          </p>
+          <div class="date-box"> 
+            <p class="date">2023.6.21(MON) ~ 25(FRI)</p>
+            <p class="place">@ National Museum of Karea, Art Center Nabi and more</p>
+          </div>
+        </div>
+        </section>
```

Comment:
as requested by the previous code review you should align all the content in this section correctly to be close to the design as much as you can. See the images below:
  your design
![image](https://user-images.githubusercontent.com/60612329/227520591-84b71102-d311-415f-85b5-d68dd6de7964.png)

template design
![image](https://user-images.githubusercontent.com/60612329/227520791-ba0ca1cb-d2ec-4915-a703-bc415481e1f2.png)